scripts/notify-suppliers-whether-application-made-for-framework.py

```diff
@@ -1,17 +1,28 @@
 #!/usr/bin/env python
-"""Email all suppliers who registered interest in applying to a framework about whether or not they made an application.
-Uses the Notify API to send emails.
+"""
+Email all suppliers who registered interest in applying to a framework about whether or not they made an application.
+
+Uses the Notify API to send emails. This script *should not* resend emails.
 
 Usage:
-    scripts/notify-suppliers-whether-application-made-for-framework.py <stage> <framework_slug>
-        <govuk_notify_api_key> [--dry-run]
+    scripts/notify-suppliers-whether-application-made-for-framework.py [options]
+         [--supplier-id=<id> ... | --supplier-ids-from=<file>]
+         <stage> <framework> <notify_api_key>
```

Comment:
This script is run by Jenkins - we should prob add in the optional arg for `--supplier-id` there.

Reply:
https://github.com/alphagov/digitalmarketplace-jenkins/pull/223